mdx-web/src/main/java/com/mx/path/model/mdx/web/filter/RequireAuthenticationFilter.java

```diff
@@ -46,6 +47,13 @@ protected final void doFilterInternal(HttpServletRequest request, HttpServletRes
       throw new UnauthorizedException("Not Authenticated", "Not Authenticated");
     }
 
+    if (PathTools.isUserRequiredPath(path)) {
+      String userId = PathTools.extractUserId(path);
+      if (userId == null || !userId.equals(Session.current().getUserId())) {
+        Session.deleteCurrent();
+        throw new UnauthorizedException("User does not match user session", "Not Authenticated");
+      }
+    }
```

Comment:
```suggestion
      String userId = PathTools.extractUserId(path);
      if (userId != null && !userId.equals(Session.current().getUserId())) {
        throw new UnauthorizedException("User does not match user session", "Not Authenticated");
      }
```

Reply:
I'm wondering if we should back off on the `Session.deleteCurrent();` until we know this works. 🤔

Reply:
I agree. How would you like to verify this working? 

- A pre-release to be verified in internally in a test environment?
- Logging statements indicating a session _would_ be deleted?